projects/canopy/src/lib/header/header.notes.ts

```diff
@@ -37,4 +37,17 @@ and in your HTML:
 | \`\`logo\`\` | A url link to the logo | string | null | Yes |
 | \`\`logoAlt\`\` | alt text to display alongside the logo | string | '2rem' | Yes |
 | \`\`logoHref\`\` | Url link if the logo is clickable | string | null | No |
+| \`\`secondLogo\`\` | A url link to the second logo | string | null | Yes |
```

Comment:
it looks like these are technically `undefined` by default not null

Reply:
@aaronBery updated thanks 👍